packages/code-du-travail-frontend/src/common/DateContenu.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class DateContenu extends React.Component {
+  static propTypes = {
+    value: PropTypes.string
+  };
+
+  render() {
+    return (
+      <div className="article__date">Publié le&nbsp;: {this.props.value}</div>
+    );
+  }
+}
+
+export { DateContenu };
```

Comment:
why not `export DateContenu` ?
A functionnal component may be enough ?

Reply:
updated ! 
I prefer named export (easier to refactor and to track error after a name change)